libs/features/dynamic-navigation.js

```diff
@@ -1,6 +1,21 @@
 import { getMetadata } from '../utils/utils.js';
 
+function checkForIgnoreValues() {
+  const dynamicNavDisableValues = getMetadata('dynamic-nav-disable');
+  if (!dynamicNavDisableValues) return false;
+
+  const metadataPairsMap = dynamicNavDisableValues.split(',').map((pair) => pair.split(';'));
+  return metadataPairsMap.reduce((rdx, pair) => {
+    const [metadataKey, metadataContent] = pair;
+    const metaTagContent = getMetadata(metadataKey.toLowerCase());
+    if (!metaTagContent
+      || metaTagContent.toLowerCase() !== metadataContent.toLowerCase()) return rdx;
+    return !rdx;
```

Comment:
I'm not certain why there's a reduce here. You seem to be flipping the accumulator every time `metaTagContent` is equal to `metadataContent`.  So if there are two cases where the `metaTagContent` is equal to `metadataContent` the function will produce `False` since it will flip twice. 
Usually `reduce` is used when we need to 'fold' (or 'reduce') a list of values (of some type) into another value (of the same or some other type). This seems less of a reduction than an imperative loop (which imo would probably make more sense to read, even though all loops have a recursive form and all recursive functions can be expressed as a fold ([source](https://www.cs.nott.ac.uk/~pszgmh/fold.pdf)))
Perhaps you were looking, instead, to do `return (!metaTagContent 
      || metaTagContent.toLowerCase() !== metadataContent.toLowerCase()) || rdx`. But in this case you have the built in `Array.prototype.some()` so you can just write 
```javascript
metadataPairsMap.some(([metadataKey, metadataContent]) => {
  const metaTagContent = getMetadata(metadataKey.toLowerCase());
  return (!metaTagContent
      || metaTagContent.toLowerCase() !== metadataContent.toLowerCase())
})
```

Reply:
Thank you. This makes perfect sense. I knew it was off, but wanted another set of eyes. I had refactored and used reduce, but `some` is much more appropriate. I appreciate it.